dev/Common/ColorConversion.cpp

```diff
@@ -268,31 +266,35 @@ winrt::hstring RgbToHex(const Rgb &rgb)
     return winrt::hstring(hexString);
 }
 
-void HexToRgba(const wstring_view& input, _Out_ Rgb *rgb, _Out_ double *alpha)
+Rgba HexToRgba(const wstring_view& input)
 {
+    Rgba result{};
     // The input always begins with a #, so we'll move past that.
```

Comment:
this local really isn't needed -- we can just fill out the struct at the return point. For example the first return below can be:
`{ Rgb(-1, -1, -1), -1 }`